### DISCUSSION TITLE
No options to select the type of account.

### DISCUSSION THREAD

**Discussion by joaochris** [sws2apps/organized-app#21] [Ideas]
Hello! Is it possible to add a definitive option to select these options, so I don't have to select every time I open the site? I liked it better when I went straight in :D

![Screenshot_2](https://user-images.githubusercontent.com/76891864/221386271-73b4a2a2-2035-4c1d-84a3-5a79ec185701.png)

**Comment by rhahao** [comment]
Hi @joaochris, thanks for your feedback. This was caused by a bug, since it should behave as you described. But it was already fixed. On your end, can you please logout from your online account first, and try to connect again? It should register again what type of account you will be using and save it as a reference for future use.

**UPDATE:**
It should no longer be needed to logout from your account. The next time you open the app, it will be the last time where you have to choose a option here. 😁

**Reply by joaochris** [reply]
Now yes!! Thanks!
